src/scripts/release.zig

```diff
@@ -27,7 +27,7 @@ const Shell = @import("../shell.zig");
 const Language = enum { dotnet, go, java, node, zig, docker };
 const LanguageSet = std.enums.EnumSet(Language);
 pub const CliArgs = struct {
-    version: []const u8,
+    run_number: u32,
```

Comment:
I think github ate my previous comment -- do we need to update `release.yml` to pass in this new flag?

Reply:
Indeed!, updated!